e2e/stack_test.go

```diff
@@ -41,12 +41,14 @@ func TestE2E(t *testing.T) {
 		t.Skip("skipping e2e tests in short mode")
 	}
 
-	const l1BlockTime = time.Second // We would want 250ms instead, but then Anvil doesn't respond to RPCs (probably a thread starving in Anvil).
+	const l1BlockTime = uint64(1) // We would want 250ms instead, but blocktimes are uints in seconds.
```

Comment:
The blocktime produced by default from the `devnetL1.json` config data is 3 seconds - we could consider removing this extra config and just going with that.

Interestingly, changing the L1 blocktime doesn't seem to change the test speed much at all. Maybe the test as-is gets all it needs from the genesis block.